src/SolutionTemplate/UnoSolutionTemplate.WinUI.net6/Mobile/UnoQuickStart.Mobile.csproj

```diff
@@ -24,6 +20,13 @@
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)'=='net6.0-macos'">10.14</SupportedOSPlatformVersion>
   	</PropertyGroup>
 
+  	<PropertyGroup Condition="'$(RuntimeIdentifier)'==''">
+		<!-- Default values for command line builds -->
+		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-ios'">iossimulator-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">maccatalyst-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-macos'">osx-x64</RuntimeIdentifier>
```

Comment:
* can't the default net6 SDK values be used ? that way `net7.0-*` support would be easier (less manual editing when updating) ?

* If not, any reason it's not defined for `net6.0-android` ? like to `android-x64`

Reply:
maybe using `TargetPlatformIdentifier` would make updating to future `net7`+ easier ?
it returns `Android`, `iOS`, `MacCatalyst` or `macOS`

Reply:
The use of RuntimeIdentifier is particularly clear to me at this time :) I know that `RuntimeIdentifier ` is automatically set for android (because apks support all of them at in a single package), but what Visual Studio does when nothing is specified is to be determined. We'll probably have to check what the command line does when unset to be certain.